src/styles/blocks/modifiers.scss

```diff
@@ -0,0 +1,17 @@
+.stars--4 .stars__item:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
The `nth-child` pseudo-class is used with a negative value. This is not valid, as the `nth-child` pseudo-class can only take positive integer values. The `-n` in the `nth-child` function is used to select elements from the end, but the value after `+` or `-` should be positive. So you should correct this.